id: PYSEC-0000-CVE-2022-30882
details: 'pyanxdns package in PyPI version 0.2 is vulnerable to code execution backdoor.
  The impact is: execute arbitrary code (remote). When installing the pyanxdns package
  of version 0.2, the request package will be installed.'
aliases:
- CVE-2022-30882
modified: '2024-11-21T14:22:59.309018Z'
published: '2022-06-08T20:15:00Z'
references:
- type: EVIDENCE
  url: https://github.com/egeback/pyanxdns/issues/1
- type: REPORT
  url: https://github.com/egeback/pyanxdns/issues/1
- type: PACKAGE
  url: https://pypi.org/project/pyanxdns/
- type: WEB
  url: http://pypi.doubanio.com/simple/request
affected:
- package:
    name: pyanxdns
    ecosystem: PyPI
    purl: pkg:pypi/pyanxdns
  ranges:
  - type: ECOSYSTEM
    events:
    - introduced: '0'
  versions:
  - 0.2.5
severity:
- type: CVSS_V3
  score: CVSS:3.1/AV:N/AC:L/PR:N/UI:N/S:U/C:H/I:H/A:H
